src/closure-externs/closure-externs.js

```diff
@@ -1069,4 +1068,12 @@ AudioWorkletNode.prototype.onprocessorerror;
 var registerProcessor = function(name, obj) {};
 var currentFrame;
 var currentTime;
-var sampleRate;
\ No newline at end of file
+var sampleRate;
+
+// These functions are declared in emcc.py but used in postamble.js; we need
+// to explicitly tell Closure Compiler about their existence since the
+// postamble is optimized independently
```

Comment:
I don't follow this. `postamble.js` is not optimized separately? Or maybe I don't understand what you mean by "postamble"?

Reply:
Gotcha, my mistake. It seems the Closure Compiler optimization is happening before the code is fully generated, or else these externs wouldn't be needed. I've updated the comment to say as much -- please let me know if you have a more accurate version in mind.